code/game/objects/items/storage/boxes.dm

```diff
@@ -1196,3 +1196,14 @@
 	new /obj/item/reagent_containers/food/snacks/grown/chili(src)
 	new /obj/item/reagent_containers/food/drinks/coffee/type2(src)
 	new /obj/item/tank/internals/emergency_oxygen(src)
+
+/obj/item/storage/box/secbat
+	name = "Secbat box"
+	desc = "Contained inside is a secbat for use with law enforcement."
+
+/obj/item/storage/box/attack_self(mob/user)

```

Comment:
I see we are pulling rabbits out of hats here. Wrong pathing.

Reply:
I'm fixing another issue that might come up with keeping it as a storage item too, so I'll incorporate this.